src/test/java/de/mediathekview/mserver/crawler/wdr/parser/WdrLetterPageDeserializerTest.java

```diff
@@ -0,0 +1,60 @@
+package de.mediathekview.mserver.crawler.wdr.parser;
+
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDTO;
+import de.mediathekview.mserver.testhelper.FileReader;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import static org.hamcrest.CoreMatchers.equalTo;
+import org.hamcrest.Matchers;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class WdrLetterPageDeserializerTest {
+  
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)